src/components/details-page/components/banner/styled.ts

```diff
@@ -55,8 +55,13 @@ const Content = styled.div`
   }
 `;
 
+const TitleWrapper = styled.div`
+  align-items: center;
+  display: flex;
+`;
+
 const Title = styled.h1`
-  display: flex inline;
+  display: inline-flex;
```

Comment:
Hvis jeg husker riktig, var det mening bak `display: flex inline;` og det er ikke det samme som `display: inline-flex;`. Du kan godt sjekke at ting fungerer fortsatt for autoritative entiteter med veldig lange titler.

Reply:
Ok, jeg kan skippe denne commit, har ikke betydning her. Så i første omgang ut bare som en feil, så jeg tok med fix. Er det gyldig å si 'display: flex inline;' ?

Reply:
Ok. Det er gyldig syntax, ja (https://developer.mozilla.org/en-US/docs/Web/CSS/display). Og denne har noen interessante eksempler: https://developer.mozilla.org/en-US/docs/Web/CSS/display/two-value_syntax_of_display.